config/pipeline.yaml

```
@@ -921,7 +921,7 @@ jobs:
   rt-tests: &rt-tests
     template: rt-tests.jinja2
     kind: job
-    nfsroot: 'https://storage.kernelci.org/images/rootfs/debian/bookworm-rt/20240313.0/{debarch}'
+    nfsroot: 'https://storage.staging.kernelci.org/rootfs/bookworm-rt/20240717.0/{debarch}'
```

Images were rebuilt to include changes from https://github.com/kernelci/kernelci-core/pull/2614, you may use here:

```suggestion
    nfsroot: 'https://storage.kernelci.org/images/rootfs/debian/bookworm-rt/20240806.0/{debarch}'
```